config/v1/types_infrastructure.go

```diff
@@ -706,6 +718,26 @@ type GCPResourceTag struct {
 	Value string `json:"value"`
 }
 
+// CloudLoadBalancerConfig contains IPs for API, API-Int and Ingress Cloud Load Balancers
+type CloudLoadBalancerConfig struct {
+	// apiLoadBalancerIPs holds API Load Balancer IPs.
+	// Could be empty for private clusters.
+	// +kubebuilder:validation:Format=ip
+	// +listType=set
+	// +optional
+	ApiLoadBalancerIPs []string `json:"apiLoadBalancerIPs,omitempty"`
```

Comment:
Can we bring back the `IP` type into config v1 that got reverted so that we have proper validation for these?